pkg/operator/ceph/cluster/cluster.go

```diff
@@ -457,7 +457,9 @@ func (c *cluster) postMonStartupActions() error {
 				// If length is one, this clearly indicates that all the mons are running the same version
 				// We are doing this because 'ceph version' might return the Ceph version that a majority of mons has but not all of them
 				// so instead of trying to active msgr2 when mons are not ready, we activate it when we believe that's the right time
-				client.EnableMessenger2(c.context, c.Namespace)
+				if err := client.EnableMessenger2(c.context, c.Namespace); err != nil {
+					logger.Errorf("%+v", err)
```

Comment:
ditto

Reply:
> Overall please name the errors instead of just print it.

For some errors I don't know the name of errors. In that cases what should I do?

Reply:
Well, each error is based on a task so just name that task.
Here, for example, you can simply say: `failed to enable messenger 2`.

Reply:
Okay